pages/team.html

```diff
@@ -37,7 +39,245 @@
+      [-80.5448914,43.4729787,0.0],
+      [-78.7923574,43.9129813,0.0],
+      [-121.72851560000001,35.960223,0.0],
+      [-122.4148138,37.785366,0.0],
+      [-73.9409065,40.7006177,0.0],
+      [-118.3982838,33.7637397,0.0],
+      [-77.6074219,43.1731354,0.0],
+      [8.7293082,47.4975352,0.0],
+      [-105.2696228,40.013811100000005,0.0],
+      [23.3239746,55.9338179,0.0],
+      [4.4961548,52.1604546,0.0],
+      [-71.0375977,42.041134,0.0],
+      [-81.3053942,28.744295300000005,0.0],
+      [-45.1229095,-22.7553666,0.0],
+      [77.6252747,12.9277742,0.0],
+      [11.5436697,48.1318547,0.0],
+      [-118.1080645,33.7421734,0.0],
+      [-105.2229309,39.7508855,0.0],
+      [19.9429321,50.0641917,0.0],
+      [-6.3061791,53.3051365,0.0],
+      [-97.7419281,30.266184099999997,0.0],
+      [-3.677845,40.4518457,0.0],
+      [-120.66580830000001,35.3026227,0.0],
+      [30.4552177,50.4472443,0.0],
+      [-80.1215744,26.643201,0.0],
+      [-87.58445560000001,41.79535500000001,0.0],
+      [-122.46545789999999,37.7464219,0.0],
+      [-10.8984375,42.2935642,0.0],
+      [2.3522219000000177,48.85661400000001,0.0],
+      [-122.3978555,37.7558973,0.0],
+      [-97.33005300000002,37.68717609999999,0.0],
+      [-95.6147432,29.703598199999995,0.0],
+      [-74.2424941,40.6207934,0.0],
+      [-118.32713980000001,33.9205867,0.0],
+      [-96.34231599999998,30.614919,0.0],
+      [-121.44856330000002,38.6910838,0.0],
+      [-121.13142700000002,38.65941,0.0],
+      [-71.114502,42.3747784,0.0]];
+
+    //Set the image for the base marker
+    var image = {
+      url: '../assets/images/icons/marker2.png',
+      size: new google.maps.Size(31, 50),
+      origin: new google.maps.Point(0, 0),
+      anchor: new google.maps.Point(3, 47)
+    };
+
+    //Set the images for the cluster markers
+    var styles = [
+      {
+        url: '../assets/images/icons/marker2_medium.png',
+        height: 75,
+        width: 47,
+        iconAnchor: [5,70],
+        textColor: '#000000',
+        textSize: 0
+      },{
+        url: '../assets/images/icons/marker2_large.png',
+        height: 100,
+        width: 62,
+        iconAnchor: [6,94],
+        textColor: '#000000',
+        textSize: 0
+      }]
+
+    //Create an array of markers
+    var markers = [];
+    for (var i = 0; i < locations.length; i++) {
+      var location = locations[i];
+      var marker = new google.maps.Marker({
+        position: {lng: location[0], lat: location[1]},
+        icon: image,
+      });
+      markers.push(marker);
+    }
+
+    //Create the marker clusterer object and apply it to the map
+    var markerCluster = new MarkerClusterer(map, markers, {gridSize: 50, maxZoom: 8, styles: styles});
+  }
+</script>
+
+<!--This uses my personal Google Developer api key. For deployment to the website please change.-->
+<script async defer
+        src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAiGG4WRCnwCREgVICtr7oMOr7Rt2UcuCo&callback=initMap">
```

Comment:
Can we change this to get it from `compile.js`? Additionally, you might want to change yours, you've just made it public.